packages/stripe_platform_interface/lib/src/stripe_platform_interface.dart

```diff
@@ -99,6 +99,7 @@ abstract class StripePlatform extends PlatformInterface {
   /// Check if either google pay or apple pay  is supported on device.
   Future<bool> isPlatformPaySupported({
     IsGooglePaySupportedParams? params,
+    PlatformPayWebPaymentRequestCreateOptions? paymentRequestOptions,
```

Comment:
I do not like the fact we need to inject a web specific method into the platform interface this should be independent.  Is this really required for just checking if platformpay is supported?

Reply:
@remonh87 thoughts on my explanation above? Do you have suggestions for a different approach here given the constraints?

Reply:
no looks to me. I noticed some conflicts in this branch but we are good to go. @jonasbark  do you want to have a look at it as well?

Reply:
Excellent. Thanks for your input. I'll get the conflicts cleared up shortly